docs/semgrep-supply-chain/overview.md

```diff
@@ -41,6 +39,8 @@ rule, the codebase scanned, and its lockfile:
 ![Relationship between a Semgrep Supply Chain rule, lockfile, CVE record, and codebase](/img/sc-reachability-analysis.png)
 _Figure 2_. Relationship between a Supply Chain rule, lockfile, CVE record, and codebase.
```

Comment:
```suggestion
_**Figure**. Relationship between a Supply Chain rule, lockfile, CVE record, and codebase._
```

Reply:
Updating the Figure caption text to latest standard. We lost the numbers because the components would sometimes have Figures on them.

I can't open suggestions the other **Figure** captions but if you reply :+1: here I will make direct commits, just let me know.

Reply:
I went through and fixed all the captions, even for the pages I didn't originally touch